### ISSUE TITLE
as.loom doesn't seem to write the scaled data into the loom object

### ISSUE THREAD

**Issue by pcm32** [satijalab/seurat#2312]
After doing scale data with Seurat and saving it to loom, reopening that file as loom, transforming into Seurat and then running the PCA module (`RunPCA`) on that object fails with the error:

```
Pulling counts matrix
No normalized data provided, not adding scaled data
No scaled data provided
Setting cluster IDs
No scaled data, not searching for dimensional reduction information
No scaled data, not searching for nearest neighbor graphs
Error in PrepDR(object = object, features = features, verbose = verbose) : 
  Data has not been scaled. Please run ScaleData and retry
Calls: RunPCA -> RunPCA.Seurat -> RunPCA -> RunPCA.Assay -> PrepDR
Execution halted
```

in contrast, serializing the scaled Seurat object through RDS and re-reading it includes the scaled data (and `runPCA` succeeds with that input).

Thanks!

**Comment by pcm32** [comment]
Apparently the scaled data is only written when specifying the `assay` in `as.loom`. Maybe that could be pointed out in the docs/vignettes.

**Comment by satijalab** [comment]
Agreed - we will update the docs if we have not already (@mojaveazure)

**Comment by JS0715** [comment]
> Apparently the scaled data is only written when specifying the `assay` in `as.loom`. Maybe that could be pointed out in the docs/vignettes.

Does this means one need to specify the assay so that the input file be normalized or scaled matrix rather than the counts?

**Comment by rsatija** [comment]
We are closing this issue now as we are no longer prioritizing support for loom conversion.

In Seurat v5, we include support for the [v5 assay class](https://satijalab.org/seurat/articles/seurat5_essential_commands.html), which can flexibly store data in a variety of formats (including disk-backed formats). We include multiple examples for disk-backed analysis using the [BPCells package from Ben Parks ](https://github.com/bnprks/BPCells).
